### PR TITLE
Refactor evaluate module

### PR DESCRIPTION
**Pull request by roshankern**
This PR is ready for review!

In this PR, the evaluate module is reformatted to include evaluations for each model (final, shuffled baseline) trained with each feature type (CP, DP, CP_and_DP) on each dataset (train, test, etc). For most of these functions, this means simply loading in the correct feature dataset with `get_X_y_data()`.

### CODE REVIEW

**Comment by roshankern** [comment]
Although this PR looks huge, most line changes are from changes in intermediate files. There are about 150 lines of code changed in this PR.

File: 3.evaluate_model/README.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
 
-In [confusion_matrices.ipynb](confusion_matrices.ipynb), we evaluate these 4 sets of predictions with a confusion matrix to see the true/false positives and negatives (see [sklearn.metrics.confusion_matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html) for more details).
+In [confusion_matrices.ipynb](confusion_matrices.ipynb), we evaluate these sets of predictions with a confusion matrix to see the true/false positives and negatives (see [sklearn.metrics.confusion_matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html) for more details).
 The confusion matrix data are saved to [confusion_matrices](evaluations/confusion_matrices).
 
-In [F1_scores.ipynb](F1_scores.ipynb), we evaluate each model (final, shuffled baseline) on each dataset (train, test, etc) to determine phenotypic and weighted f1 scores.
+In [F1_scores.ipynb](F1_scores.ipynb), we evaluate each model (final, shuffled baseline) trained with each feature type (CP, DP, CP_and_DP) on each dataset (train, test, etc) to determine phenotypic and weighted f1 scores.
```
Comment (d33bs) [inline 3.evaluate_model/README.md:13]: Are these feature types defined anywhere in the documentation? If not, consider providing their definition and/or linking to it.

File: utils/evaluate_utils.py [hunk 2]
```diff
@@ -81,7 +83,7 @@ def class_PR_curves(
         axs[ax_x, ax_y].set(xlabel="Recall", ylabel="Precision")
 
         ax_x += 1
-        if ax_x == 4:
+        if ax_x == 3:
```
Comment (d33bs) [inline utils/evaluate_utils.py:88]: Consider adding some comments here and around this line to describe what's happening.

File: 3.evaluate_model/scripts/nbconverted/F1_scores.py
Comment (d33bs) [inline 3.evaluate_model/scripts/nbconverted/F1_scores.py:1]: While I was looking through these files I was thinking about automatic formatting which might help some of the longer or multi-line blocks. This isn't a requirement of review so much as a suggestion towards increasing development velocity. If it helps, and in context with the IDE you're most comfortable with, consider using [`black[jupyter]`](https://black.readthedocs.io/en/stable/getting_started.html#installation) or [`jupyterlab_code_formatter`](https://github.com/ryantam626/jupyterlab_code_formatter) which help to format code within notebooks using Python black, etc.

**Review by d33bs — APPROVED** [review]
Nice work! I thought the changes here looked good to me and mostly had minor suggestions for your consideration. Please don't hesitate to let me know if you have any questions.

**Review by axiomcura — APPROVED** [review]
Nice job. Just had a small comment of the sorting of the directories :)

File: 3.evaluate_model/README.md (hunk 1 above)
Comment (roshankern) [reply 3.evaluate_model/README.md:13]: Added to documentation in https://github.com/WayScience/phenotypic_profiling_model/pull/21/commits/7a024fd3f9feb6e0e3a69296cea221d48c5e52c0

File: utils/evaluate_utils.py (hunk 2 above)
Comment (roshankern) [reply utils/evaluate_utils.py:88]: Added comments in https://github.com/WayScience/phenotypic_profiling_model/pull/21/commits/7a024fd3f9feb6e0e3a69296cea221d48c5e52c0

File: 3.evaluate_model/scripts/nbconverted/F1_scores.py
Comment (roshankern) [reply 3.evaluate_model/scripts/nbconverted/F1_scores.py:1]: Although I was using `black` to format `.py` files, I never realized I could do `format notebook` with Python. Thanks for the tip!

**Comment by roshankern** [comment]
Thank you both for the review! @axiomcura your suggestions have been implemented in https://github.com/WayScience/phenotypic_profiling_model/pull/21/commits/1f311922a03200cdcfa74a54e9d9c2048d64d1f1
